### DISCUSSION TITLE
Pain point: Speed on Windows

### DISCUSSION THREAD

**Discussion by CheshireSwift** [orogene/orogene#17] [Pain Points]
Related to #9, but tends to be more than just the size of files. All JS package managers seem slow on Windows. `pnpm` is the only one that's ever been a tolerable speed, but seems to not work on half my projects (as in, the dependencies are just plain missing).

As acknowledged in #9, Windows is far from an ideal dev environment, but sometimes it's what you've got!

**Comment by zkat** [comment]
My main development os right now is Windows :) I definitely intend to prioritize performance on it!
